introspection-engine/introspection-engine-tests/tests/native_types/postgres.rs

```diff
@@ -108,45 +110,77 @@ async fn native_type_columns_feature_on(api: &TestApi) -> crate::TestResult {
 
     Ok(())
 }
-//
```

Comment:
Should we remove the comments before merging?

Reply:
There are no more comments, the test is active now.